File: DistributionFrontApp/src/app/Models/Device.model.ts
export class Device{

    id: number;
    name: string;
    type: string;
    longitude: number;
    latitude: number;
    address: string;

    constructor(id: number, name: string, type: string, longitude: number, latitude: number, address: string){
        
        this.id = id;
        this.name = name;
        this.type = type;
        this.longitude = longitude;
        this.latitude = latitude;
        this.address = address;

    }

}

File: DistributionSmartEnergyBackApp/DistributionSmartEnergyBackApp/Models/EntityModels/DeviceModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DistributionSmartEnergyBackApp.Models.EntityModels
{
    public class DeviceModel
    {
        private string name;
        private string type;
        private long id;
        private string address;
        private double longitude;
        private double latitude;
        public string documentId { get; set; }

        public long Id
        {
            get
            {
                return id;
            }
            set
            {
                id = value;
            }
        }

        public string Type
        {
            get
            {
                return type;
            }
            set
            {
                type = value;
            }
        }

        public string Name
        {
            get 
            { 
                return name;
            }
            set
            {
                name = value;
            }
        }

        public string Address
        {
            get
            {
                return address;
            }
            set
            {
                address = value;
            }
        }

        public double Longitude
        {
            get
            {
                return longitude;
            }
            set
            {
                longitude = value;
            }

        }

        public double Latitude
        {
            get
            {
                return latitude;
            }
            set
            {
                latitude = value;
            }
        }

        
    }
}
